app/api/kb/articles/route.ts

```
@@ -0,0 +1,139 @@
+    if (!session?.user) {
+      return new ChatSDKError('unauthorized:api').toResponse();
+    }
+
+    const body = await request.json();
+    const { id, title, content, summary, tags, categoryId } = body;
+
+    if (!id) {
+      return new ChatSDKError('bad_request:api').toResponse();
+    }
+
+    let embedding;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)